### ISSUE TITLE
Progress Wiped/Store Closed

### ISSUE THREAD

**Issue by Catfish1245** [0xf4b1/hearthstone-linux#49]
I was confused to see that after I gave it a login token that my progress in the game had been wiped. I double checked on my mobile device and I had still had all the packs and progress I had made. I still have my friends, just not my game progress. I decided just to restart, however it's worth noting in case it happens to somebody else.

I was also curious if there was a way to emulate the missing resources in regards to the shop so that it can be used.

**Comment by 0xf4b1** [comment]
I've never experienced a loss in progress, did you maybe choose another region? then you would start from beginning as if a new account is created if you never played on that region

**Comment by Catfish1245** [comment]
It appears I may have it in the wrong region. Is there a way I can fix that?

**Comment by 0xf4b1** [comment]
yeah, you can directly set it in the `client.config` and change the line `Aurora.Env=REGION.actual.battle.net` and replace REGION with the correct one :)

**Comment by 0xf4b1** [comment]
it should be `enUS` and make sure to edit the file in the hearthstone folder, not the one in the project repo (it should already contain something else than `REGION`)

**Comment by Catfish1245** [comment]
Thank you
